client/src/entrypoints/images/image-chooser-modal.js

```diff
@@ -1,3 +1,7 @@
+import $ from 'jquery';
+import { initTabs } from '../../includes/tabs';
+
+/* eslint-disable */
```

Comment:
No, we can’t just disable all of ESlint for the whole file 😉 

Add this file’s path and `document-chooser-modal` to the array at https://github.com/wagtail/wagtail/blob/main/.eslintrc.js#L115 and this should be all that’s needed for now.

Reply:
hahah genius 🤓